Adds hipaa user session timeout behavior [DOC-743] (#5570)
segmentio/segment-docs: src/privacy/hipaa-eligible-segment.md

4. On the **Turn off data encryption?** popup, click **Confirm**.

> success ""
> Disabling the data encryption setting does not decrypt existing data, but does prevent any future data from being encrypted.

> ### User session timeouts

Reviewer: Making this an h2 to set it apart from the data encryption section directly above it

```suggestion
## User session timeouts
```

Author: Updated in 83405229ed244bba36d26506243e74536f70eb7e

> Segment will automatically log off users with access to HIPAA eligible workspaces after 15 minutes of inactivity, and they will
> need to re-authenticate.

Reviewer: ```suggestion
Segment automatically logs out all users with access to HIPAA eligible workspaces after 15 minutes of inactivity.
```

Author: Updated in 83405229ed244bba36d26506243e74536f70eb7e
